lib/nfs_common.pm

```diff
@@ -214,8 +215,8 @@ sub config_service {
     try_nfsv2();
 
     prepare_exports($rw, $ro);
-
-    my $module_name = y2_module_consoletest::yast2_console_exec(yast2_module => 'nfs-server');
+    my $y2_opts = is_pvm_hmc() ? "--ncurses" : "";
```

Comment:
I think it is more simple without "()"

Reply:
This is on purpose, it will show clearly function call.